python/ray/serve/scripts.py

```diff
@@ -195,3 +281,54 @@ def info(address: str):
         print(json.dumps(response.json(), indent=4))
     else:
         log_failed_request(response, address)
+
+
+@cli.command(
+    help="[Experimental] Get your Serve application's status.",
+    hidden=True,
+)
+@click.option(
+    "--address",
+    "-a",
+    default=os.environ.get("RAY_ADDRESS", "http://localhost:8265"),
+    required=False,
+    type=str,
+    help='Address of the Ray dashboard to query. For example, "http://localhost:8265".',
+)
+def status(address: str):
+    full_address_path = f"{address}/api/serve/deployments/status"
+    response = requests.get(full_address_path)
+    if response.status_code == 200:
+        print(json.dumps(response.json(), indent=4))
```

Comment:
cli_logger?

Reply:
I used `print` here, so the output could be read directly as JSON. This was useful for testing because we can read the output as a dictionary. Also users can programmatically read the output without any string parsing.

If we use `cli_logger`, the JSON gets prepended with `2022-03-01 11:46:17,206	INFO scripts.py:282 -- `, which means it can't be read directly. Do you think we should stick with `print` or switch to the logger?

Reply:
Hmm, I see, we should definitely not prefix with timestamps. I think @architkulkarni used a flag to avoid adding this formatting in his PR -- could you comment on this? Otherwise we can stick with `print`, I'd just like to keep all output going through the same interface.

Reply:
You can pass in `_format=False` to `cli_logger.print()`, but I think that does an unrelated kind of formatting.  It just does the `f`-string type formatting where you can pass in args and they get substituted wherever you have curly braces.

Reply:
I see, I think @architkulkarni is right. The `no_format` flag doesn't remove the timestamp, so I'll stick with `print` for now.

Reply:
OK